superset-frontend/src/explore/constants.ts

```diff
@@ -28,49 +28,80 @@ export const AGGREGATES = {
 };
 export const AGGREGATES_OPTIONS = Object.values(AGGREGATES);
 
-export const OPERATORS = {
-  '==': '==',
-  '!=': '!=',
-  '>': '>',
-  '<': '<',
-  '>=': '>=',
-  '<=': '<=',
-  IN: 'IN',
-  'NOT IN': 'NOT IN',
-  ILIKE: 'ILIKE',
-  LIKE: 'LIKE',
-  REGEX: 'REGEX',
-  'IS NOT NULL': 'IS NOT NULL',
-  'IS NULL': 'IS NULL',
-  'LATEST PARTITION': 'LATEST PARTITION',
-  'IS TRUE': 'IS TRUE',
-  'IS FALSE': 'IS FALSE',
+export enum Operators {
+  EQUALS = 1,
```

Comment:
if we're using this as an id, then for ease of debugging, i think i'd prefer using
```
EQUALS = 'EQUALS',
NOT_EQUALS = 'NOT_EQUALS',
etc.
```